File: Prácticas/P3/Java/civitas/src/civitas/OperacionInmobiliaria.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

/**
 *
 * @author Jose Padial
 */
public class OperacionInmobiliaria {
    private int numPropiedad;
    private OperacionesInmobiliarias gestion;

    public OperacionInmobiliaria(OperacionesInmobiliarias gest, int ip) {
        this.numPropiedad = ip;
        this.gestion = gest;
    }

    public int getNumPropiedad() {
        return numPropiedad;
    }

    public OperacionesInmobiliarias getGestion() {
        return gestion;
    }
}

File: Prácticas/P3/Ruby/civitas/pkg/civitas-0.0.1/lib/operacion_inmobiliaria.rb
# encoding: utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module Civitas
  class OperacionInmobiliaria
     attr_reader :num_propiedades, :gestion
    def initialize(gest,ip)
      @num_propiedades = ip
      @gestion = gest
    end
  end
end